coffea/processor/executor.py

```diff
@@ -901,20 +1228,14 @@ class Runner:
     savemetrics: bool = False
     mmap: bool = False
     schema: Optional[schemas.BaseSchema] = schemas.BaseSchema
-    cachestrategy: Optional[Union[Literal["dask-worker"], Callable[..., MutableMapping]]] = None  # fmt: skip
+    cachestrategy: Optional[
+        Union[Literal["dask-worker"], Callable[..., MutableMapping]]
+    ] = None  # fmt: skip
     processor_compression: int = 1
     use_skyhook: Optional[bool] = False
+    skyhook_options: Optional[Dict] = field(default_factory=dict)
     format: str = "root"
 
-    @staticmethod
-    def read_coffea_config():
```

Comment:
Why is this deleted?

Reply:
Hmm, something weird happened when I updated the branch